build.gradle

```diff
@@ -331,12 +331,8 @@ tasks.register 'allJavadoc', Javadoc, {
             languageVersion = JavaLanguageVersion.of(11)
         }
         jdoc.inputs.file javaDocOverviewLocation
-        if (project.findProperty('includeJavadoc') == 'true') {
-            jdoc.inputs.files project.project(':javadoc').tasks.getByName('compileJava').outputs.files
-        }
         jdoc.options.encoding = 'UTF-8'
         jdoc.options.docletpath = [new File("$rootDir/javadoc/build/classes/java/main")]
-        jdoc.options.doclet = 'io.deephaven.javadoc.IncludeDoclet'
         jdoc.options.tags = ['Include:a:', 'Exclude:a:', 'IncludeAll:a:', 'apiNote', 'implNote']
```

Comment:
all the tags w/ `:` are not needed anymore.  Not sure about apiNote and implNote